assets/js/app.js

```diff
@@ -12,22 +12,39 @@ import "../css/app.scss"
 //     import {Socket} from "phoenix"
 //     import socket from "./socket"
 //
-import "phoenix_html"
-import {Socket} from "phoenix"
-import NProgress from "nprogress"
-import {LiveSocket} from "phoenix_live_view"
+import "phoenix_html";
+import { Socket } from "phoenix";
+import NProgress from "nprogress";
+import { LiveSocket } from "phoenix_live_view";
 
-let csrfToken = document.querySelector("meta[name='csrf-token']").getAttribute("content")
-let liveSocket = new LiveSocket("/live", Socket, {params: {_csrf_token: csrfToken}})
+let Hooks = {};
+Hooks.LedColor = {
```

Comment:
defining the [phoenix hook]( https://hexdocs.pm/phoenix_live_view/js-interop.html#client-hooks) that listens on mount and changes the color of the led to the one provided in the data attribute described below.

Later we can move this into `updated()` callback then update the html content 

We can use an interval genserver to try and achieve the rainbow effect on our live view